processor/resourcedetectionprocessor/internal/gcp/metadata.yaml

```diff
@@ -50,4 +50,12 @@ resource_attributes:
   k8s.cluster.name:
     description: The k8s.cluster.name
     type: string
-    enabled: true
\ No newline at end of file
+    enabled: true
+  gcp.gce.instance.name:
+    description: The name of the GCE instance.
+    type: string
+    enabled: true
+  gcp.gce.instance.hostname:
+    description: The hostname of the GCE instance.
+    type: string
+    enabled: true
```

Comment:
As I mentioned in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23681#discussion_r1245604655, new resource attributes should be added as optional initially. Otherwise, it's considered a breaking change. If we want to make them enabled by default, we will need to apply a warning to notify users in advance after they are added as optional. We'll get an option to add such warnings in `metadata.yaml` once https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/19174 is resolved. I've done some prerequisite work for that, and will add the warnings option next week.

Reply:
Sorry, got confused because we reverted that with https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23681#discussion_r1245608301, but I forgot that's because Cloud Run Jobs were totally new. I updated this to disable the new GCE attributes by default